packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js

```diff
@@ -0,0 +1,84 @@
+// import { expect } from 'chai';
```

Comment:
Why are these tests completely commented out? 🤔

Reply:
They aren't commented out in PFE, but there are no  notes here to say why they've been removed.

Reply:
Intended to fix, lint, etc. in #533, but better to include in this PR, thanks!